### PR TITLE
Advise against new bytecode instructions

### PR DESCRIPTION
**Pull request by ncik-roberts**
Add a comment that urges caution when adding a new bytecode instruction. (It's not impossible &mdash; we've done it &mdash; but it shouldn't be done lightly.)

### CODE REVIEW

**Review by mshinwell — APPROVED** [review]
The comment is fine but I think you might have broken a horrible script.
OK to merge once that is fixed

**Comment by ncik-roberts** [comment]
A good reminder that one should always wait for CI to pass, even if you've just added a comment.

**Pull request merged**
